### ISSUE TITLE
How to test float / int?

### ISSUE THREAD

**Issue by dcworldwide** [zhaoyao91/meteor-validate#1]
I think isString(), isDate(), isFloat() is better then isType(type<string>). Saves typing and avoids errors.

**Comment by zhaoyao91** [comment]
good idea. but there are some problems with isFloat or isInt. as i know, js only has the type number,  no float or int(though it may be implemented as double). how to tell a number to be int or float?

**Comment by zhaoyao91** [comment]
I can't figure out how to check float or int as js only has type number. Otherwise, I have added all the type -checking validations in version 2.

**Comment by dcworldwide** [comment]
Neat nice work. I'm not sure how to test floats / ints either.

**Comment by zhaoyao91** [comment]
So I want to keep this issue open to welcome ideas for this problem.
